modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -473,7 +475,16 @@ protected void validateField(
 		builder.withInstanceId(fieldInstanceId);
 
 		if (!valid) {
-			String errorMessage = ddmFormFieldValidation.getErrorMessage();
+			LocalizedValue errorMessageLocalizedValue =
```

Comment:
Sometimes you name the variables as `errorMessage` and sometimes not, but I understand that, to keep consistency, you should name the variables always as `errorMessage`. After all, the type of the `errorMessage`, `LocalizedValue`, is already explicit in the class `DDMFormFieldValidation`.

Reply:
Got it, I changed this part. Now the `LocalizedValue` typed objects are the ones called `errorMessage`.